src/modules/invoices/database/queries/invoices.repository.ts

```
@@ -1,20 +1,25 @@
 import {
   eq, and, isNull, sql, desc,
 } from 'drizzle-orm';
+import type {
+  InvoiceWithRelations,
+  InvoiceListFilters,
+} from '@/modules/invoices/types/invoices.types';
```

ESLint

[error] `@/modules/invoices/types/invoices.types` type import should occur after type import of `@/modules/invoices/database/schema`

(import/order)